src/components/logger/logger.js

```diff
@@ -2,6 +2,7 @@ import 'typeface-montserrat/index.css';
 import React from 'react';
 import PropTypes from 'prop-types';
 import styled from 'styled-components';
+import { isEqual } from 'lodash';
```

Comment:
better to import it like import isEqual from 'lodash/isEqual'; to decrease bundle size

Reply:
I just replaced it by old school `===`